docs/usage.md

```diff
@@ -49,4 +49,38 @@ To be done.
 
 ## I'm the release manager
 
-To be done.
+> NOTE: This instructions are for the 0.2.0 milestone (when ready). As more features and automation is added the process will become simpler.
+
+### Preparing the changelog
+
+* Wait for the last BC of the release. If another BC is generated after that or a patch version for a previous minor is released, you might need to restart the process.
+* Create a branch from the commit of the BC.
+* From the root folder of the repository run:
+
+```
+$ elastic-agent-changelog-tool build --version x.y.z --owner <owner> --repo <repo>
```

Comment:
@endorama @LucianPy if there any default value for `owner` ? If not, should we make `elastic` the default value?

Reply:
Yes, default owner is already `elastic`: https://github.com/elastic/elastic-agent-changelog-tool/blob/7c056a963218f80cd2a57f16f0f4bd21c55d2b53/cmd/root.go#L11

The default `repo` is `beats`, we may want to update it to `elastic-agent`.